### DISCUSSION TITLE
How should you configure it in order to work properly?

### DISCUSSION THREAD

**Discussion by basings** [RandomNinjaAtk/arr-scripts#114] [General]
As an example, I'd like to usr the band "AC/DC". 
After setting the desired quality, specifying the arl token and setting a root folder, lidarr is ready to start. According to the logs, the fuzzy search algorithm does not seem to work reliably. What do I need to set in order for the script to work?

**Comment by RandomNinjaAtk** [comment]
I welcome PR with changes that can improve the search/matching logic.... It works as well as its currently designed (which is pretty decent, my opinion)... If you can improve it, PR welcome!
